test/libsolidity/syntaxTests/using/using_global_library_for_interface.sol

```diff
@@ -0,0 +1,6 @@
+interface I {}
```

Comment:
Not sure I understand the naming convention for these tests. Shouldn't this one be `global_library_for_interface.sol`?